src/gtest/test_libzendoo.cpp

```diff
@@ -25,6 +25,13 @@ extern unsigned char ReverseBitsInByte(unsigned char input);
 
 using namespace blockchain_test_utils;
 
+// Keep them aligned with those defined in CCTPlib!
```

Comment:
Can we leverage getCBG() method to insert a static assert in order to make sure constant values in test_libzendoo.cpp are aligned with those in` sidechainTxsCommitmentBuilder.h`? Or at least add a check within the test execution?